include/Reader/readerir.h

```diff
@@ -558,6 +560,12 @@ class GenIR : public ReaderBase {
 
   EHRegion *rgnAllocateRegion() override;
   EHRegionList *rgnAllocateRegionList() override;
+  void setDebugLocation(uint32_t CurrOffset, uint32_t IsCall) override;
```

Comment:
Andy's suggestion is better: I hadn't really realized that `IsCall == 0` meant that the call originated from an empty operand stack.